projects/plugins/jetpack/_inc/client/at-a-glance/backup-upgrade/style.scss

```
@@ -0,0 +1,45 @@
+.jp-dash-upgrade-backup {
+    margin-bottom: 0px;
+    padding-top: 3rem;
+    padding-bottom: 3rem;
+
+    &__popup {
+        position: absolute;
+        top: -10px;
+        left: 0;
+        right: 0;
+        margin: 0 auto;
+        width: 337px;
+        background: rgba(255, 255, 255, 0.86);
+        border: 1px solid #DCDCDE;
```

In general, the Jetpack Design team prefers we use CSS color variables instead of explicit hex or RGBA. In this case, it looks like we could replace `#DCDCDE` like so:

```suggestion
        border: 1px solid var( --jp-gray );
```

---

Thank you.

---

Fixed, thank you